packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testResizeWithZoomBack.st

```diff
@@ -0,0 +1,10 @@
+zooming
+testResizeWithZoomBack
+	
+	| zoomLevel scrollPane zoomBack |
+	zoomLevel := 0.6.
+	zoomBack := 1.0.
+	scrollPane := SScrollPane forMorph: instance.
+	instance zoomTo: zoomLevel.
+	instance zoomTo: zoomBack.
+	self assert: (instance boundsInWorld area >= scrollPane boundsInWorld area).
```

Comment:
@AlexImmer is there a simple way to find out the exact size of the new bounds instead of just using a greater than?
otherwise we leave it this way